cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,43 @@
-describe('', () => {
-  before(() => {
 
-  });
+describe('Article', () => {
+  let user;
+  let articleTest;
+
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
 
-  it('', () => {
+    cy.task('generateArticle').then((generateArticle) => {
+      articleTest = generateArticle;
+    });
 
+    cy.visit('');
   });
 
-  it('', () => {
+  it('The article should be created', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.visit('/editor');
+    cy.get('[placeholder="Article Title"]').type(articleTest.title);
+    cy.get('[placeholder="What\'s this article about?"]')
+      .type(articleTest.description);
+    cy.get('[placeholder="Write your article (in markdown)"]')
+      .type(articleTest.body);
+    cy.contains('.btn', 'Publish Article').click();
+    cy.contains('.container', articleTest.title).should('exist');
+  });
+
+  it('The article should be deleted', () => {
+    cy.login(user.email, user.username, user.password);
```

Comment:
There's no need to login again in the 'it' block for deleting the article if the user session is maintained. If it's necessary due to the session not being maintained, consider maintaining the session to speed up the tests.